modules/mapper-extras/src/test/resources/org/elasticsearch/index/mapper/filebeat-6.0.template.json

```diff
@@ -73,6 +73,10 @@
                     "location": {
                       "type": "geo_point"
                     },
+                    "region_iso_code": {
+                      "ignore_above": 1024,
```

Comment:
I don't know why `ignore_above` is set on this or the other iso code field, but it should not be needed (for city/region name it might make sense since they could be particularly long depending on the language).

Reply:
Yes, you're right. I will change this.